### DISCUSSION TITLE
how can I keep current selection (highlight) and move the cursor?

### DISCUSSION THREAD

**Discussion by wgqimut** [helix-editor/helix#10999] [Q&A] [answered]
when I navigate a function, I want to know how a variable is used, I usually highlight them (by `<space>h`) and see the locations it used, but if a function is long enough, I need to move the page down, then I lost my previous highlight (selections), in vscode I just highlight them and use the mouse to scroll, Does any way to do it in helix?

**Comment by kirawi** [comment, marked as answer]
Enter view mode, `z`. You can lock view mode with `Z` until you exit with `escape`. However, the behavior is limited right now, see https://github.com/helix-editor/helix/issues/2134
